doc-content/enterprise-only/openshift/operator-createapp-proc.adoc

```diff
@@ -0,0 +1,125 @@
++
+. Add lines to the YAML source to modify the environment:
+** The following snippet adds an immutable {KIE_SERVER} pod that builds a service from source. You must add at least one copy of this snippet when creating an immutable environment.
++
+[subs="attributes,verbatim,macros"]
+----
+  objects:
+    servers:
+      - build:
+          kieServerContainerDeployment: <deployment>
+          gitSource:
+            uri: <url>
+            reference: <branch>
+            contextDir: <directory>
+----
++
+Replace the following values:
++
+*** `<deployment>`: The identifying information of the decision service (KJAR file) that is built from your source. The format is: `<containerId>=<groupId>:<artifactId>:<version>`. You can provide two or more KJAR files using the `|` separator, for example: `containerId=groupId:artifactId:version|c2=g2:a2:v2`. The Maven build process must produce all these files from the source in the Git repository.
+*** `<url>`: The URL for the Git repository that contains the source for your decision service.
+*** `<branch>`: The branch in the Git repository.
+*** `<directory>`: The path to the source within the project downloaded from the Git repository (??? where is the artifact directory ???)
++
+** The following snippet sets the number of {KIE_SERVERS} that are managed by {CENTRAL} or {CENTRAL} Monitoring in your environment: 
++
+[subs="attributes,verbatim,macros"]
+----
+  objects:
+    server:
+      # Number of server deloyments (defaults are 0, 1, or 2 for different environment types)
+      deployments: 2
+      spec:
+        # Environment variables that are set for all the KIE servers
+        env:
+          - name: MY_VAR
+            value: "example"
+        # Override default memory limits for all the KIE servers
+        resources:
+          memory:
+            limits: 2Gi
+----
++
+** The following snippet removes Smart Router from your environment: ???
+** The following snippet sets the password for the administrator user (`admin`):
++
+[subs="attributes,verbatim,macros"]
+----
+  commonConfig:
+    adminPassword: password
+----
++
+** The following snippet sets the URL to an external Maven repository. In a production environment, the Maven repository must contain all the services that you need to deploy.
++
+[subs="attributes,verbatim,macros"]
+----
+  commonConfig:
+    mavenRepo: https://maven.example.com
+----
++
+** The following snippet sets up RH-SSO authentication. You must ensure that the realm and clients exist in your RH-SSO deployment. The number of listed servers must be the same as the number of actual {KIE_SERVERS} in your environment. (??? what about the option where deployment auto-creates the clients - do operators support this?)
++
+[subs="attributes,verbatim,macros"]
+----
+  auth:
+    sso:
+      url: https://rh-sso.example.com:8443
+      realm: rhpam
+      clients:
```

Comment:
This has all been moved into the individual object sections... see -
https://github.com/kiegroup/kie-cloud-operator/blob/1.0.x/deploy/examples/auth_sso.yaml